symphony-bdk-core/src/main/java/com/symphony/bdk/core/activity/command/HelpCommand.java

```diff
@@ -68,4 +68,17 @@ protected ActivityInfo info() {
         .name(HELP_COMMAND)
         .description(DEFAULT_DESCRIPTION);
   }
+
+  @Override
+  public boolean equals(Object o) {
```

Comment:
if equals is defined, hashcode must be too (https://stackoverflow.com/questions/2265503/why-do-i-need-to-override-the-equals-and-hashcode-methods-in-java)